sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.CSharp9.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel;
+
+TopLevelLocalFunction("literal", "literal"); //Noncompliant
+                                             //Noncompliant@-1
```

Comment:
Isn't the second diagnostic a FP? As far as I can tell only the first parameter is decorated. For `Property2` there is no issue raised.

Reply:
The RSPEC states that the rule is trigger also for
>The parameter or property name contains "Text", "Message", or "Caption".

As the [original test](https://github.com/SonarSource/sonar-dotnet/blob/2850f216baf6a41e1cd7478918abc61a3eea73d2/sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.cs#L23) tests this explicit list, I wanted to demonstrate one attribute-based and one name-based scenario in C# 9 context.

So it should be a true positive.